Misc/NEWS.d/next/Library/2022-03-13-08-52-58.bpo-46998.cHh-9O.rst

```diff
@@ -0,0 +1 @@
+Allow subclassing of :class:`typing.Any`
```

Comment:
```suggestion
Allow subclassing of :class:`typing.Any`. Patch by Shantanu Jain.
```
Seems like you're already in `Misc/ACKS`.

Reply:
Yeah, I've made a couple dozen PRs to CPython at this point :-)